vendor/github.com/containerd/fifo/utils.go

```diff
@@ -14,7 +14,7 @@
    limitations under the License.
 */
 
-package sys
+package fifo
```

Comment:
Let me comment on this change as it can be easily missed; this file has now been removed from `sys/`, and instead  using the one from `containerd/fifo`

<img width="592" alt="Screenshot 2021-04-19 at 09 31 24" src="https://user-images.githubusercontent.com/1804568/115198156-0ae14f80-a0f2-11eb-839f-947117e8775f.png">